### DISCUSSION TITLE
Amazon IAP

### DISCUSSION THREAD

**Discussion by anishamalde** [hyochan/react-native-iap#2259] [Show and tell]
Hi,
I wrote a guideline on how you can get the sample working with Amazon IAP. It also works with Expo Go. You can have a look here: https://dev.to/amazonappdev/react-native-iap-one-package-to-rule-them-all-4f0n
Please feel free to drop me any questions and big kudos to the team for maintaining this awesome package.

**Comment by hyochan** [comment]
Thanks for the additional context, and thanks to @anishamalde for sharing the Amazon IAP guide back in 2023 — it’s a great resource and definitely helpful for anyone experimenting with Amazon IAP.

That said, while community guides and past examples exist, Amazon IAP / Fire OS is not officially supported or actively maintained in react-native-iap v14+ at the moment. This is mainly due to the lack of a proper testing environment and ongoing resources on my side.

For more clarity on the current stance, I’ve summarized it here:
https://github.com/hyochan/expo-iap/discussions/239

If someone from the community (or Amazon) is willing to help with testing, maintenance, or official support, I’m absolutely open to revisiting this.

For now, I’ll close this discussion to avoid confusion around official support. The shared guide remains a useful reference for community-driven setups. Thanks for understanding!
